conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ConjureUndertowRuntime.java

```diff
@@ -68,22 +72,34 @@ public AuthorizationExtractor auth() {
         return auth;
     }
 
+    @Override
+    public AsyncRequestProcessing async() {
+        return async;
+    }
+
     public static final class Builder {
 
+        private Duration asyncTimeout = Duration.ofMinutes(3);
```

Comment:
Do we want to increase this to match `ClientConfigurations.DEFAULT_READ_TIMEOUT` (5 minutes)? I chose 3 because it's a relatively low number, and it will be harder to decrease this moving forward than it is to increase. I also considered using 2 minutes based on https://github.com/palantir/conjure-java-runtime/pull/966

5 minutes is the safest bet to avoid breaking the proxy use case, but we may leverage this to reduce call times.

Thoughts?